### PR TITLE
Add/fix darwin support for various virt-manager dependencies

### PR DESCRIPTION
**Pull request by mikroskeem**
###### Motivation for this change

I am planning to get virt-manager up and running on Darwin.

Supersedes bunch of changes in stale-ish PRs:
- #113306
- #128767

Depends on:
- #152371 

###### Things done

- Built on platform(s)
  - [ ] x86_64-linux
  - [ ] aarch64-linux
  - [x] x86_64-darwin
  - [x] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [ ] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [22.05 Release Notes (or backporting 21.11 Release notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2205-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
  - [ ] (Release notes changes) Ran `nixos/doc/manual/md-to-db.sh` to update generated release notes
- [ ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

**Comment by mikroskeem** [comment]
All actions have consequences, this PR won't avoid them either. Will add additional fixes reported by `nixpkgs-review pr 152348`.

Right now I'm seeing gtkd -> libgda -> gcc6, which is not going to work out.

**Comment by mikroskeem** [comment]
Seems like 133b9618f8a58a83060301f37a2ba138d91d2b59 broke GTK-3 on aarch64-darwin.

**Comment by mikroskeem** [comment]
After ~8 hours of building... never going to do staging branch builds on my laptop ever again lol.

Result of `nixpkgs-review pr 152348` run on aarch64-darwin [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>13 packages built:</summary>
  <ul>
    <li>geany-with-vte</li>
    <li>germinal</li>
    <li>gtk-vnc</li>
    <li>gtkd</li>
    <li>libgda</li>
    <li>libosinfo</li>
    <li>libvirt-glib</li>
    <li>osinfo-db</li>
    <li>osinfo-db-tools</li>
    <li>spice-gtk</li>
    <li>termite (termite-unwrapped)</li>
    <li>usbredir</li>
    <li>vte</li>
  </ul>
</details>

**Comment by mikroskeem** [comment]
@NixOS/darwin-maintainers

**Comment by toonn** [comment]
Hmm, nixpkgs-review failed for me:

```
building '/nix/store/53jl7cn13wk51b42wxbgyqkm8v6m620s-bootstrap-stage2-stdenv-darwin.drv'...
building '/nix/store/x3pmi0qp8bbj07ynpr125h0cvfysvnsg-bootstrap-stage2-stdenv-darwin.drv'...
error: output '/nix/store/kknyif7z0py8h6gybp47fdzp5dgib90h-bootstrap-stage2-stdenv-darwin' is not allowed to refer to the following paths:
         /nix/store/5p5pncjap0kax6jzsh5jxq2nz5h75b7y-zstd-1.5.1
         /nix/store/m4vzwxx1qvvbpslisljzz35xx0n5ay3w-libunistring-0.9.10
         /nix/store/x7x5s166m6dn6vanr0zcwm9pkrprvsbg-libidn2-2.3.2
error: output '/nix/store/ynngj9vlnl6q5bd5v63yrhpvx4vwq48n-bootstrap-stage2-stdenv-darwin' is not allowed to refer to the following paths:
         /nix/store/5p5pncjap0kax6jzsh5jxq2nz5h75b7y-zstd-1.5.1
         /nix/store/m4vzwxx1qvvbpslisljzz35xx0n5ay3w-libunistring-0.9.10
         /nix/store/x7x5s166m6dn6vanr0zcwm9pkrprvsbg-libidn2-2.3.2
error: 1 dependencies of derivation '/nix/store/q7ypyd32hdss0637v51vb4dp3l1mbm81-adv_cmds-119.drv' failed to build
error: 1 dependencies of derivation '/nix/store/sf98rj1zplhwghc7a370xhdgq8f9pxbg-bash-5.1-p12.drv' failed to build
error: build of '/nix/store/48frjw3q6fdpf4wfzi1d944ypxa8whx9-stdenv-darwin.drv', '/nix/store/i21njz3vb4ljfh84mj26a4giidawhg75-bash-interactive-5.1-p12.drv', '/nix/store/sf98rj1zplhwghc7a370xhdgq8f9pxbg-bash-5.1-p12.drv' failed
$ git worktree prune
```

However, @alyssais mentioned in the Darwin room that Darwin stdenv is broken on staging so this may (I suspect it is) just be a consequence of that.

**Comment by alyssais** [comment]
Yeah, that's the same error I saw for other PRs.

**Comment by mikroskeem** [comment]
Bump

**Comment by uri-canva** [comment]
Running the build on my laptops, so far so good, but it's going to take a while.

**Comment by uri-canva** [comment]
`x86_64-darwin` running Monterey completed:
```
error: builder for '/nix/store/hywwzznlra216pfyyyawp92zg1yphq99-libxkbcommon-1.3.1.drv' failed with exit code 1;
       last 10 log lines:
       > Expected Fail:      0
       > Fail:               1
       > Unexpected Pass:    0
       > Skipped:            1
       > Timeout:            0
       >
       > Full log written to /private/tmp/nix-build-libxkbcommon-1.3.1.drv-0/libxkbcommon-1.3.1/build/meson-logs/testlog.txt
       > FAILED: meson-test
       > /nix/store/1mmpb3lm7zpi7fs9qmcdkycdqhgq6zpa-meson-0.60.3/bin/meson test --no-rebuild --print-errorlogs
       > ninja: build stopped: subcommand failed.
       For full logs, run 'nix log /nix/store/hywwzznlra216pfyyyawp92zg1yphq99-libxkbcommon-1.3.1.drv'.
error: builder for '/nix/store/snydhfkdazvb2wsgzc0lmwmiavsh6v54-orc-0.4.32.drv' failed with exit code 1;
       last 10 log lines:
       > Expected Fail:      0
       > Fail:               0
       > Unexpected Pass:    0
       > Skipped:            0
       > Timeout:            1
       >
       > Full log written to /private/tmp/nix-build-orc-0.4.32.drv-0/orc-0.4.32/build/meson-logs/testlog.txt
       > FAILED: meson-test
       > /nix/store/1mmpb3lm7zpi7fs9qmcdkycdqhgq6zpa-meson-0.60.3/bin/meson test --no-rebuild --print-errorlogs
       > ninja: build stopped: subcommand failed.
       For full logs, run 'nix log /nix/store/snydhfkdazvb2wsgzc0lmwmiavsh6v54-orc-0.4.32.drv'.
error: builder for '/nix/store/w4fc7fs9pqgb6l864i80wpxrmc0kdhz6-graphite2-1.3.14.drv' failed with exit code 2;
       last 10 log lines:
       > 84/84 Test #84: vm-test-direct-threading .........   Passed    5.30 sec
       >
       > 99% tests passed, 1 tests failed out of 84
       >
       > Total Test time (real) =   7.12 sec
       >
       > The following tests FAILED:
       >      80 - nametabletest (Timeout)
       > Errors while running CTest
       > make: *** [Makefile:71: test] Error 8
       For full logs, run 'nix log /nix/store/w4fc7fs9pqgb6l864i80wpxrmc0kdhz6-graphite2-1.3.14.drv'.
error: 1 dependencies of derivation '/nix/store/3d07x3qyk5avx9pf7qszfyc7di62mrm3-harfbuzz-3.2.0.drv' failed to build
error: 1 dependencies of derivation '/nix/store/cq1mhn2slsyk4hwqxamdrrzd2xl5wn41-harfbuzz-3.2.0.drv' failed to build
error: 1 dependencies of derivation '/nix/store/w12n5r4yjv60l14g3k7cy1d4yb691dfm-libass-0.15.2.drv' failed to build
error: 1 dependencies of derivation '/nix/store/aqm543njn1vcvqi1fxn5q4ggrw0jk79h-pango-1.50.3.drv' failed to build
error: 2 dependencies of derivation '/nix/store/ddqd8ny5xyaclxxp5cdzxd6b70lcklck-gtk+3-3.24.31.drv' failed to build
error: 1 dependencies of derivation '/nix/store/fzr1zyzm51d5xivhahgi9v7hm4yj5x9p-appstream-glib-0.7.18.drv' failed to build
error: 2 dependencies of derivation '/nix/store/a6z2y26si15v0xbrbm97j1gxafmilmcl-gtksourceview-4.8.2.drv' failed to build
error: 1 dependencies of derivation '/nix/store/82cgyxbc0jxm4r45rg2mxg887bd9aydp-libgda-5.2.10.drv' failed to build
error: 1 dependencies of derivation '/nix/store/7vf5yrrzsy33nk12a512xppml4l5bx6r-libpeas-1.30.0.drv' failed to build
error: 2 dependencies of derivation '/nix/store/mf392gy6xz8km9fplvx4hbipzjlbljhp-vte-0.64.2.drv' failed to build
error: 2 dependencies of derivation '/nix/store/qpb8hn5g981shyq0vx6zhl7i4z2w2k7i-vte-0.64.2.drv' failed to build
error: 2 dependencies of derivation '/nix/store/47v22hhjl6gw9barqp1ky37vm3rk204s-gst-plugins-base-1.18.5.drv' failed to build
error: 2 dependencies of derivation '/nix/store/8xrhx7gl71xc61qfn09ffzgjidmbjd3q-gst-plugins-good-1.18.5.drv' failed to build
error: 3 dependencies of derivation '/nix/store/wpwyy224aph2pp5rkp12xq0lnfh2q664-spice-gtk-0.37.drv' failed to build
error: 1 dependencies of derivation '/nix/store/cr5yvr22c65836qjk2zfca3lc6i0gswn-gtk-vnc-1.3.0.drv' failed to build
error: 1 dependencies of derivation '/nix/store/kxn00fi9rffaadx8zdhnbf9xcvqrqy5j-ffmpeg-4.4.1.drv' failed to build
error: 2 dependencies of derivation '/nix/store/qzfx6mnf6zg9sblsczdhn56zhz400f9y-opencv-4.5.4.drv' failed to build
error: 1 dependencies of derivation '/nix/store/8swp13fv34bs6r7myp3i60x9nmrl9yqc-librsvg-2.52.4.drv' failed to build
error: 6 dependencies of derivation '/nix/store/9rly96nb7bxy6kc06wgszcd5bwslmrfn-gst-plugins-bad-1.18.5.drv' failed to build
error: 2 dependencies of derivation '/nix/store/zd2czd7z34bz3b287pbrfnfpfpm4ad7y-hook.drv' failed to build
error: 4 dependencies of derivation '/nix/store/jkxkmi7pl5y18pq57xl4wnx3ynzw6p6z-germinal-26.drv' failed to build
error: 9 dependencies of derivation '/nix/store/i4ill45ha4cd5163n66p13s2p745q32j-gtkd-3.9.0.drv' failed to build
error: 3 dependencies of derivation '/nix/store/naqfy3iagfqbpw6z25bh74irl5a30ckg-termite-15.drv' failed to build
error: 6 dependencies of derivation '/nix/store/45fz9035rw0bv1sr2cfdqbck4d3504bv-review-shell.drv' failed to build

Link to currently reviewing PR:
https://github.com/NixOS/nixpkgs/pull/152348

2 packages marked as broken and skipped:
mate.mate-terminal virt-manager-qt

6 packages failed to build:
germinal gtk-vnc gtkd spice-gtk termite vte

5 packages built:
libosinfo libvirt-glib osinfo-db osinfo-db-tools usbredir

error: build log of '/nix/store/jkxkmi7pl5y18pq57xl4wnx3ynzw6p6z-germinal-26.drv' is not available
error: build log of '/nix/store/xmml809msi8i27gsq230aq5qn60ibc10-germinal-26' is not available
error: build log of '/nix/store/cr5yvr22c65836qjk2zfca3lc6i0gswn-gtk-vnc-1.3.0.drv' is not available
error: build log of '/nix/store/1if87v487jg7f7yxcisz25y1mij63mzk-gtk-vnc-1.3.0' is not available
error: build log of '/nix/store/i4ill45ha4cd5163n66p13s2p745q32j-gtkd-3.9.0.drv' is not available
error: build log of '/nix/store/h555irl2yqzmgr1j06lhpj06345hd8vc-gtkd-3.9.0' is not available
```

**Comment by uri-canva** [comment]
I disabled the tests for the three packages that were failing: `libxkbcommon`, `orc` and `graphite2` and the remaining packages built correctly. The build on the M1 laptop is still going.

**Comment by uri-canva** [comment]
Build on `aarch64-darwin` fails because of `rustc`, but I can't see it on hydra so it might have already been broken to begin with.

**Comment by mikroskeem** [comment]
I heard that tzdata build also fails while reviewing this PR with sandbox enabled. This is not the rabbit hole I expected to come into... 😫I'll look into other build regressions (gtkd, vte etc.).---- On Fri, 21 Jan 2022 04:25:15 +0200  Uri ***@***.***> wrote ---- 
Build on aarch64-darwin fails because of rustc, but I can't see it on hydra so it might have already been broken to begin with.

**Comment by uri-canva** [comment]
I think if you disable the checks for darwin you can merge it, it's ok if it's broken on `aarch64-darwin` since these packages aren't currently building there anyway, and the change makes progress on `x86_64-darwin`. Once it's merged it will make testing and iterating over the changes easier since most of them will be cached.

**Comment by mikroskeem** [comment]
Yesterday's build:

Result of `nixpkgs-review pr 152348` run on aarch64-darwin [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>2 packages failed to build:</summary>
  <ul>
    <li>gtkd</li>
    <li>spice-gtk</li>
  </ul>
</details>
<details>
  <summary>11 packages built:</summary>
  <ul>
    <li>geany-with-vte</li>
    <li>germinal</li>
    <li>gtk-vnc</li>
    <li>kermit-terminal</li>
    <li>libosinfo</li>
    <li>libvirt-glib</li>
    <li>osinfo-db</li>
    <li>osinfo-db-tools</li>
    <li>termite (termite-unwrapped)</li>
    <li>usbredir</li>
    <li>vte</li>
  </ul>
</details>

Also got following build errors: https://gist.github.com/mikroskeem/702cbc7c7de67630c561dbc5122df5c2

**Comment by mikroskeem** [comment]
Bump, need help with testing this.

**Comment by siraben** [comment]
@mikroskeem there's another merge conflict

**Comment by mikroskeem** [comment]
Fixed conflicts, should be good to go now.

**Comment by ereslibre** [comment]
Thanks for your efforts @mikroskeem!, the build is failing on me due to `mesonFlags` being duplicated in the `vte` derivation.

```
error: attribute 'mesonFlags' already defined at /Users/ereslibre/.cache/nixpkgs-review/pr-152348-2/nixpkgs/pkgs/development/libraries/vte/default.nix:79:3
```

**Comment by mikroskeem** [comment]
Fixed that as well now.

**Comment by ereslibre** [comment]
Result of `nixpkgs-review pr 152348` run on x86_64-darwin [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>2 packages marked as broken and skipped:</summary>
  <ul>
    <li>mate.mate-terminal</li>
    <li>virt-manager-qt</li>
  </ul>
</details>
<details>
  <summary>5 packages failed to build:</summary>
  <ul>
    <li>germinal</li>
    <li>gtkd</li>
    <li>libvirt-glib</li>
    <li>spice-gtk</li>
    <li>termite (termite-unwrapped)</li>
  </ul>
</details>
<details>
  <summary>7 packages built:</summary>
  <ul>
    <li>gtk-vnc</li>
    <li>kermit-terminal</li>
    <li>libosinfo</li>
    <li>osinfo-db</li>
    <li>osinfo-db-tools</li>
    <li>usbredir</li>
    <li>vte (xfce.vte)</li>
  </ul>
</details>

**Comment by mikroskeem** [comment]
Build logs @ereslibre ?

**Comment by ereslibre** [comment]
https://gist.github.com/ereslibre/a96651a5ecc0c1b46b71b64691fe0af5

**Comment by ereslibre** [comment]
Getting https://gist.github.com/ereslibre/eeb11cf9264fba6096f4f869ba254c1b now.

When I have some time I would like to help further than just running the build, but I cannot do that right now.

**Comment by mikroskeem** [comment]
Can you grab me `nix log /nix/store/2gll3hmiqam863d1nwq2jhs4h28apy9i-libvirt-8.1.0.drv`?

**Comment by ereslibre** [comment]
> Can you grab me nix log /nix/store/2gll3hmiqam863d1nwq2jhs4h28apy9i-libvirt-8.1.0.drv?

https://gist.github.com/ereslibre/2ab02b7bbf9bcc8c2fa873267751653a

**Comment by mikroskeem** [comment]
VTE build fixed, rebased PR enough on staging so I didn't step into other unrelated build failures.

**Comment by ereslibre** [comment]
Result of `nixpkgs-review pr 152348` run on x86_64-darwin [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>13 packages failed to build:</summary>
  <ul>
    <li>geany-with-vte</li>
    <li>germinal</li>
    <li>gtk-vnc</li>
    <li>gtkd</li>
    <li>kermit-terminal</li>
    <li>libosinfo</li>
    <li>libvirt-glib</li>
    <li>osinfo-db</li>
    <li>osinfo-db-tools</li>
    <li>spice-gtk</li>
    <li>termite (termite-unwrapped)</li>
    <li>usbredir</li>
    <li>vte</li>
  </ul>
</details>

**Comment by mikroskeem** [comment]
Building this using `nixpkgs-review pr ...` fails for me as well, so seems like things in staging/master seem to be broken atm.

Last commit fixes libvirt build.

**Comment by mikroskeem** [comment]
Result of `nixpkgs-review` run on aarch64-darwin [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>2 packages marked as broken and skipped:</summary>
  <ul>
    <li>perl532Packages.SysVirt</li>
    <li>perl534Packages.SysVirt</li>
  </ul>
</details>
<details>
  <summary>2 packages failed to build:</summary>
  <ul>
    <li>gtkd</li>
    <li>ocamlPackages.ocaml_libvirt</li>
  </ul>
</details>
<details>
  <summary>20 packages built:</summary>
  <ul>
    <li>docker-machine-kvm</li>
    <li>geany-with-vte</li>
    <li>germinal</li>
    <li>gtk-vnc</li>
    <li>kermit-terminal</li>
    <li>libosinfo</li>
    <li>libvirt</li>
    <li>libvirt-glib</li>
    <li>nixops_unstable</li>
    <li>osinfo-db</li>
    <li>osinfo-db-tools</li>
    <li>python310Packages.libvirt</li>
    <li>python39Packages.libvirt</li>
    <li>rubyPackages.ruby-libvirt</li>
    <li>rubyPackages_3_0.ruby-libvirt</li>
    <li>rubyPackages_3_1.ruby-libvirt</li>
    <li>spice-gtk</li>
    <li>termite (termite-unwrapped)</li>
    <li>usbredir</li>
    <li>vte</li>
  </ul>
</details>

gtkd build fails because [ldc / ldc-bootstrap does not build](https://gist.github.com/mikroskeem/71792b1b2a6b286777e18d0ac7ab7dea). Been a case since I opened this PR I believe.

ocamlPackages.ocaml_libvirt [build log](https://gist.github.com/mikroskeem/dc441cafeddbe63da7ef5f0dce700698)

File: pkgs/development/libraries/libvirt/default.nix [hunk 1]
```diff
@@ -141,6 +141,7 @@ stdenv.mkDerivation rec {
     sed -i '0,/qemuxml2argvtest/{/qemuxml2argvtest/d;}' tests/meson.build
   '' + optionalString isDarwin ''
     sed -i '/qemucapabilitiestest/d' tests/meson.build
+    sed -i '/vircryptotest/d' tests/meson.build
```
Comment (SuperSandro2000) [inline pkgs/development/libraries/libvirt/default.nix:144]:
```suggestion
    sed -i tests/meson.build \
      -e '/qemucapabilitiestest/d' \
      -e '/vircryptotest/d' tests/meson.build
```

File: pkgs/development/libraries/vte/default.nix [hunk 2]
```diff
@@ -45,6 +45,12 @@ stdenv.mkDerivation rec {
       url = "https://git.alpinelinux.org/aports/plain/community/vte3/fix-W_EXITCODE.patch?id=4d35c076ce77bfac7655f60c4c3e4c86933ab7dd";
       sha256 = "FkVyhsM0mRUzZmS2Gh172oqwcfXv6PyD6IEgjBhy2uU=";
     })
+  ] ++ lib.optionals stdenv.isDarwin [
```
Comment (SuperSandro2000) [inline pkgs/development/libraries/vte/default.nix:48]:
```suggestion
```

**Comment by ereslibre** [comment]
`asgiref` is not passing tests on Mac OS. I have created https://github.com/django/asgiref/pull/321 and confirmed that it builds and tests pass now.

Depending on upstream, we can wait until `3.5.1` is tagged and released, or we can carry a patch for `asgiref` until that happens. @SuperSandro2000, given you are the maintainer of `asgiref`, thoughts?

**Comment by ereslibre** [comment]
`asgiref` is not passing tests on Mac OS. I have created https://github.com/django/asgiref/pull/321 and confirmed that it builds and tests pass now.

Depending on upstream, we can wait until `3.5.1` is tagged and released, or we can carry a patch for `asgiref` until that happens. @SuperSandro2000, given you are the maintainer of `asgiref`, thoughts?

**Comment by SuperSandro2000** [comment]
Fetching https://github.com/django/asgiref/commit/31541da6e68c1961f00d206b8386aee6df76d8b6.patch is fine. We want to get finished with this.

**Comment by siraben** [comment]
Result of `nixpkgs-review pr 152348` run on aarch64-darwin [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>2 packages marked as broken and skipped:</summary>
  <ul>
    <li>perl532Packages.SysVirt</li>
    <li>perl534Packages.SysVirt</li>
  </ul>
</details>
<details>
  <summary>22 packages failed to build:</summary>
  <ul>
    <li>docker-machine-kvm</li>
    <li>geany-with-vte</li>
    <li>germinal</li>
    <li>gtk-vnc</li>
    <li>gtkd</li>
    <li>kermit-terminal</li>
    <li>libosinfo</li>
    <li>libvirt</li>
    <li>libvirt-glib</li>
    <li>nixops_unstable</li>
    <li>ocamlPackages.ocaml_libvirt</li>
    <li>osinfo-db</li>
    <li>osinfo-db-tools</li>
    <li>python310Packages.libvirt</li>
    <li>python39Packages.libvirt</li>
    <li>rubyPackages.ruby-libvirt</li>
    <li>rubyPackages_3_0.ruby-libvirt</li>
    <li>rubyPackages_3_1.ruby-libvirt</li>
    <li>spice-gtk</li>
    <li>termite (termite-unwrapped)</li>
    <li>usbredir</li>
    <li>vte</li>
  </ul>
</details>

**Comment by ereslibre** [comment]
@mikroskeem you can cherry-pick https://github.com/ereslibre/nixpkgs/commit/7a6d6749d1e575ca5aa0f65c9d0637a7cdd34e5b so that asgiref tests pass.

**Comment by ereslibre** [comment]
Now it's rustc the one failing to build: https://gist.github.com/ereslibre/22f219d9856d182c1ee9c16691932ba6

**Comment by ereslibre** [comment]
Note: https://github.com/ereslibre/nixpkgs/commit/7a6d6749d1e575ca5aa0f65c9d0637a7cdd34e5b included patch is not up to date. The commit that made it into the `asgiref` project was https://github.com/django/asgiref/commit/d451a724c93043b623e83e7f86743bbcd9a05c45. This is the commit that will be included sometime in asgiref 3.5.1 when it's tagged.

**Comment by mikroskeem** [comment]
I give up

**Comment by ereslibre** [comment]
> I give up

Thanks @mikroskeem for all your efforts. Really appreciated.

**Pull request closed without merging**